plop-templates/component/package.json.hbs

```diff
@@ -0,0 +1,23 @@
+{
+ 	"name": "@ag.ds-next/{{packageName}}",
+ 	"version": "0.0.1",
+ 	"homepage": "https://steelthreads.github.io/agds-next/packages/{{category}}/{{packageName}}",
+ 	"license": "MIT",
+ 	"main": "dist/ag.ds-next-{{packageName}}.cjs.js",
+  "module": "dist/ag.ds-next-{{packageName}}.esm.js",
+ 	"dependencies": {
+		"@babel/runtime": "^7.4.5"
+ 	},
+ 	"devDependencies": {
```

Comment:
Small nit: Can we move the order of `dependencies`, `devDependencies` and `peerDependencies` to be consistent with existing packages?